src/quick-view.scss

```diff
@@ -0,0 +1,100 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-quick-view;
+
+$fd-quick-view-avatar-indent: 0.75rem;
+$fd-quick-view-title-bottom-indent: 0.5rem;
+
+.#{$block} {
+  @include fd-reset();
+
+  min-width: 20rem;
+  max-width: 30rem;
+
+  &__content {
+    @include fd-reset();
+
+    padding: 1rem;
+  }
+
+  &__header {
+    @include fd-reset();
+
+    > *:first-child {
+      border-top-right-radius: $fd-popover-border-radius;
+      border-top-left-radius: $fd-popover-border-radius;
+    }
+  }
```

Comment:
Could you use in examples bar with `--header` modifier, instead of having separated `__header` element?

Reply:
changed